src/components/spinner/Spinner.tsx

```diff
@@ -54,10 +54,7 @@ export const Spinner = forwardRef<HTMLSpanElement, SpinnerProps>((props, ref) =>
 
   return (
     <>
-      <SpinnerWrapper
-        className={wrapperClassName}
-        inline={children === undefined}
-        isLoading={isLoading}>
```

Comment:
You might need to remove the prop definition from SpinnerWrapper as well

Reply:
<img width="632" alt="image" src="https://github.com/user-attachments/assets/aa393afd-d690-4662-9d24-7840685959e3">
isLoading isn't needed here now